n/nlohmann_json/nlohmann_json-3.10.4-GCCcore-12.2.0.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'CMakeMake'
+
+name = 'nlohmann_json'
+version = '3.10.4'
```

Comment:
we have `3.11.2` already in EasyBuild with same toolchain, can that be used instead?